mobile-app/lib/ui/views/learn/landing/landing_viewmodel.dart

```diff
@@ -184,20 +185,26 @@ class LearnLandingViewModel extends BaseViewModel {
         );
       }
 
-      return buttonData;
+      Future<List<SuperBlockButtonData>> btnList = Future.value(buttonData);
+
+      return btnList;
```

Comment:
Why convert this to a future value?

Reply:
I think I went for a different approach, which I abandoned and never changed back.. I will push a fix.